src/api/spec/controllers/webui/configuration_controller_spec.rb

```diff
@@ -5,19 +5,19 @@
   let(:admin_user) { create(:admin_user) }
 
   describe 'GET #index' do
-    it { should use_before_action(:require_admin) }
+    it { is_expected.to use_before_action(:require_admin) }
```

Comment:
@ChrisBr @hennevogel what is the action running in those "describe" cases? I mean, I don't see any "get" running here... isn't it?